trinity/extensibility/component_manager.py

```
@@ -84,16 +85,16 @@ def __init__(self,
             # a little bit of extra try/finally structure here to produce good
             # logging messages about the component lifecycle.
             try:
-                async with AsyncExitStack() as stack:
-                    self.logger.info(
-                        "Starting components: %s",
-                        '/'.join(component.name for component in enabled_components),
-                    )
-                    # Concurrently start the components.
-                    await asyncio.gather(*(
-                        stack.enter_async_context(run_component(component))
-                        for component in enabled_components
-                    ))
+                self.logger.info(
+                    "Starting components: %s",
+                    '/'.join(component.name for component in enabled_components),
+                )
+                context_managers = [run_component(component) for component in enabled_components]
+                # XXX: aiotools seems to be a pretty young project, so we may want to just lift
```

I think I'm supportive of this.  The code is simple enough but the fact that it simply ignores exceptions rather than re-raising them is not something I think we want.